modules/GenericDataParser/src/server/generic_data_parser.R

```diff
@@ -223,6 +223,77 @@ validateMetaData <- function(metaData, configList, username, formatSettings = li
 
   return(list(validatedMetaData=validatedMetaData, duplicateExperimentNamesAllowed=duplicateExperimentNamesAllowed, useExisting=useExisting))
 }
+
+
+saveIncomingEndpointData <- function(codes, codeKind) {
+  # Saves incoming column data into their codetable / data dictionaries
+  # set up for endpoint manager data, only looking at "column name", "column units", "concentration units", "time units" for now... 
+  # We are going off just starting to just take one input columnName to save
```

Comment:
nit: Is this comment line still relevant?

Reply:
Good catch!